helpers/argumentParsers.go

```diff
@@ -621,3 +623,95 @@ func ParseBPFCmd(cmd int32) string {
 
 	return res
 }
+
+// UnameRelease gets the version string of the current running kernel
+func UnameRelease() string {
+	var uname syscall.Utsname
+	if err := syscall.Uname(&uname); err != nil {
+		return ""
+	}
+	var buf [65]byte
+	for i, b := range uname.Release {
+		buf[i] = byte(b)
+	}
+	ver := string(buf[:])
+	if i := strings.Index(ver, "\x00"); i != -1 {
+		ver = ver[:i]
+	}
+	return ver
+}
+
+func ParseKernelReadFileId(id int32) (string, error) {
```

Comment:
I feel this needs some explanation on why it should exist, despite being just a helper.

Reply:
no problem, i'll a few lines to explain.

Reply:
i've added a small explanation - is that good enough or should i elaborate more?

Reply:
This is not the place for such a function.
The helpers package is intended for a user of libbpfgo, which might need to parse known types used by different Linux system calls. This enum, however, is an internal kernel type, which is too specific to be included here

Reply:
this enum is used in the LSM hook `security_kernel_read_file`  which the user may use. that's why i thought this is a good place for this helper. see: https://elixir.bootlin.com/linux/v5.14/source/fs/kernel_read_file.c#L20

Reply:
I had the same concerns as @yanivagman thus my question to 'explain' it further.

Reply:
ok, so i'll remove this helper out of libbpgo.